### DISCUSSION TITLE
How to suppress / hide "Download the React DevTools for a better development experience"

### DISCUSSION THREAD

**Discussion by williamlmao** [vercel/next.js#65467] [Help]
### Summary

<img width="1168" alt="image" src="https://github.com/vercel/next.js/assets/55903529/a2085b2d-48b3-4b25-aee0-6ed08d053d45">

How do I get rid of this error? I already have react devtools installed. When searching I found some threads for vite/vanilla react but didn't find any nextjs specific instructions.

**Comment by icyJoseph** [comment]
Maybe you need to update the React DevTools again? or perhaps, close the tab and reopen? is the React DevTools extension active?
